rake/cldr/puller.rb

```diff
@@ -45,11 +45,10 @@ def default_branch_sha
         end
 
         def normalize_components(components)
-          components = components.presence
```

Comment:
This line depends on a rails method, but it's not necessary because `components` comes from `ENV["COMPONENTS"]`, which is `nil` anyways if that env var is unset (not an empty string).